app/views/policy_dashboard/_policy_status_widget.html.erb

```diff
@@ -1,12 +1,15 @@
 <div id='status-table'>
   <h4 class="header"><%= _('Hosts Breakdown') %></h4>
   <ul>
-    <%= status_link _('Compliant with the policy'), :compliant_hosts, arf_reports_path(:search => "comply_with = \"#{@policy.name}\"") %>
-    <%= status_link _('Not compliant with the policy'), :incompliant_hosts, arf_reports_path(:search => "not_comply_with = \"#{@policy.name}\"") %>
-    <%= status_link _('Inconclusive results'), :inconclusive_hosts, arf_reports_path(:search => " inconclusive_with = \"#{@policy.name}\"") %>
+    <%= status_link _('Compliant with the policy'), :compliant_hosts, hosts_path(:search => "comply_with = \"#{@policy.name}\"") %>
+    <%= status_link _('Not compliant with the policy'), :incompliant_hosts, hosts_path(:search => "not_comply_with = \"#{@policy.name}\"") %>
+    <%= status_link _('Inconclusive results'), :inconclusive_hosts, hosts_path(:search => " inconclusive_with = \"#{@policy.name}\"") %>
     <%= status_link _('Never audited'), :report_missing, hosts_path(:search => "compliance_report_missing_for = \"#{@policy.name}\"") %>
     <h4 class="total">
-      <%= link_to(_("Total hosts: %s") % @report[:assigned_hosts], hosts_path(:search => "compliance_policy = \"#{@policy.name}\"")) %>
+      <%= link_to(_("Total hosts with policy: %s") % @report[:assigned_hosts], hosts_path(:search => "compliance_policy = \"#{@policy.name}\"")) %>
+    </h4>
+    <h4>
+      <%= link_to(_("Total hosts with reports where policy was unassigned: %s") % @report[:unassigned_hosts], hosts_path(:search => "removed_from_policy = \"#{@policy.name}\"")) %>
```

Comment:
How about calling it `Hosts no longer assigned` with tooltip explaining what happened. It's clear what it's assigned to as we're on policy dashboard page.

Reply:
I added the tooltip and changed the wording.

![hosts-link](https://user-images.githubusercontent.com/2664090/57521483-b5567280-7320-11e9-909a-933ee33807f2.png)